language/stdlib/modules/LibraAccount.move

```diff
@@ -212,17 +217,7 @@ module LibraAccount {
         // Check that the `to_deposit` coin is non-zero
         let deposit_value = Libra::value(&to_deposit);
         assert(deposit_value > 0, 7);
-        let travel_rule_limit_microlibra = DualAttestationLimit::get_cur_microlibra_limit();
-        // travel rule only applies for payments over a threshold
-        let approx_lbr_microlibra_value = Libra::approx_lbr_for_value<Token>(deposit_value);
-        let above_threshold = approx_lbr_microlibra_value >= travel_rule_limit_microlibra;
-        // travel rule only applies if the sender and recipient are both VASPs
-        let both_vasps = VASP::is_vasp(payer) && VASP::is_vasp(payee);
-        if (above_threshold &&
-            both_vasps &&
-            // travel rule does not apply for intra-VASP transactions
-            VASP::parent_address(payer) != VASP::parent_address(payee)
-        ) {
+        if (travel_rule_applies<Token>(payer, payee, deposit_value)) {
```

Comment:
Nit: can we call this `dual_attestation_applies`? Although the old comments only mentioned the travel rule, the dual attestation protocol is/will be used for both the travel rule and for other purposes.